example/ios/Podfile.lock

```diff
@@ -70,9 +70,9 @@ PODS:
     - FlipperKit/FlipperKitNetworkPlugin
   - fmt (6.2.1)
   - glog (0.3.5)
-  - hermes-engine (0.72.4):
-    - hermes-engine/Pre-built (= 0.72.4)
-  - hermes-engine/Pre-built (0.72.4)
+  - hermes-engine (0.72.12):
+    - hermes-engine/Pre-built (= 0.72.12)
+  - hermes-engine/Pre-built (0.72.12)
```

Comment:
Why hermes-engine is 0.72.12 instead 0.72.14? Have you run a `pod install` after upgrade?

Reply:
Good catch, I run it but somehow hermes-engine did not get updated.